### PR TITLE
2018 state of Haskell survey results

### PR DESCRIPTION
**Pull request by tfausak**
Still very much a work in progress. I'm hoping to get this published tomorrow, but we'll see about that.

### CODE REVIEW

File: _posts/2018-11-16-2018-state-of-haskell-survey-results.markdown [hunk 1]
```diff
@@ -0,0 +1,1562 @@
+---
+title: 2018 state of Haskell survey results
+---
+
+The [2018 state of Haskell survey][1] opened at the start of November and few a couple days ago.
```
Comment (snoyberg) [inline _posts/2018-11-16-2018-state-of-haskell-survey-results.markdown]:
```suggestion
The [2018 state of Haskell survey][1] opened at the start of November and closed a couple days ago.
```

**Comment by tfausak** [comment]
It seems that someone may have submitted a bunch of bogus results. Discussion is happening on the haskell-community mailing list: https://mail.haskell.org/pipermail/haskell-community/2018-November/000358.html

**Comment by tfausak** [comment]
I updated my script to ignore the bogus submissions. Of the 5,096 responses, 3,417 were ignored, leaving 1,679 legitimate responses. You can see the exact script that I used to prune the responses and produce the charts and tables here: https://gist.github.com/tfausak/cc381ddd0ddd8304cc3eae6250966f55/0dd7415b657c2f361bbbbf305b1db869803a2c40

I will not be providing a CSV with all 5,096 responses due to the likelihood of misuse. If someone is interested in the full data set, they can download it from the Git history here: https://github.com/tfausak/tfausak.github.io/blob/94b537aafe259ffb6093c239aeafb8e7a40dd22a/static/pages/2018-11-16-2018-state-of-haskell-survey-results.csv.zip

This announcement post will only include results from the 1,679 legitimate responses. If further problems are found with this limited data set, it is easy enough to re-run the script to produce new artifacts. In fact, I encourage others to do just that in order to verify that I have not misrepresented the results. 

I need to re-write the introduction to account for these shenanigans, but otherwise this post is ready to go.

**Comment by tfausak** [comment]
Ok, the script has been updated again to remove even more bogus responses. Hopefully the third time's a charm here.

**Pull request merged**